examples/pyinstaller/hook-cefpython3.py

```diff
@@ -133,7 +137,8 @@ def get_cefpython3_datas():
         # CEF binaries and datas
         extension = os.path.splitext(filename)[1]
         if extension in [".exe", ".dll", ".pak", ".dat", ".bin",".txt", ".so", ".plist"] \
-            or filename.lower().startswith("license"):
+                or filename.lower().startswith("license") \
+            :
```

Comment:
Does this pass pep8?

Reply:
I've just made some fixes to conform PEP8.